examples/tutorials/descriptors_filter.ipynb

```diff
@@ -0,0 +1,287 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "73bfb65e",
+   "metadata": {},
+   "source": [
+    "Load the aiida profile and code:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "79098139",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from aiida import load_profile\n",
+    "load_profile()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "3bc2cf3b",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from aiida_mlip.data.model import ModelData\n",
+    "uri = \"https://github.com/stfc/janus-core/raw/main/tests/models/mace_mp_small.model\"\n",
+    "model = ModelData.from_uri(uri, architecture=\"mace_mp\", cache_dir=\"mlips\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "ae62816f",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from aiida.orm import load_code\n",
+    "janus_code = load_code(\"janus@localhost\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "65fbe939",
+   "metadata": {},
+   "source": [
+    "Inputs should include the model, code, metadata, and any other keyword arguments expected by the calculation we are running:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "1795dd41",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from aiida.orm import Str, Float, Bool\n",
+    "inputs = {\n",
+    "    \"code\": janus_code,\n",
+    "    \"model\": model,\n",
+    "    \"arch\": Str(model.architecture),\n",
+    "    \"precision\": Str(\"float64\"),\n",
+    "    \"device\": Str(\"cpu\"),\n",
+    "    \"fmax\": Float(0.1), \n",
+    "    \"opt_cell_lengths\": Bool(False), \n",
+    "    \"opt_cell_fully\": Bool(True), \n",
+    "    \"metadata\": {\"options\": {\"resources\": {\"num_machines\": 1}}},\n",
+    "    }"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1d661593",
+   "metadata": {},
+   "source": [
+    "We now load the calculations we want to run:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "0e4c78c9",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from aiida.plugins import CalculationFactory\n",
+    "\n",
+    "geomoptCalc = CalculationFactory(\"mlip.opt\")\n",
+    "descriptorsCalc = CalculationFactory(\"mlip.descriptors\")\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "37df1f81",
+   "metadata": {},
+   "source": [
+    "Now we can create our WorkGraph. This includes passing in the inputs, looping through and running the calculations on each structure."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "86036116",
+   "metadata": {},
+   "source": [
+    "In the loop we can call each structure; \n",
+    "* Run geomopt calculation \n",
+    "* Pass the output file into and run descriptors calculation \n",
```

Comment:
We should be clearer what the output (file) is that we want to pass onwards, as there are several outputs.